notebooks/270-sound-generation/270-sound-generation.ipynb

```diff
@@ -0,0 +1,953 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.        cleanup_torchscript_cache()
</pre><p>Make sure the garbage is collected for every step to clean up the memory immediately</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
done